tools/rw-heatmaps/rw-benchmark.sh

```diff
@@ -2,7 +2,7 @@
 
 #set -x
 
-RATIO_LIST="1/128 1/8 1/4 1/2 2/1 4/1 8/1 128/1"
+RATIO_LIST="${RATIO_LIST:-1/128 1/8 1/4 1/2 2/1 4/1 8/1 128/1}"
```

Comment:
This is good thanks, suggest we extend it to other key dimensions below like `RUN_COUNT` and `KEY_SIZE` at the same time. Most of the script parameters should be tunable via emvironment vars imo.

Reply:
Hi @jmhbnz, as I was working on enabling the rest of the arguments. I realized that the script allows setting some of the arguments (mostly not performance-related). My question is, do you prefer execution arguments or environment variables? I don't have a strong preference, although sometimes it's easier to use environment variables for automated jobs, and arguments are easier for end-user execution. We could support both but I'm not sure if that would be an overkill.

Ref: https://github.com/etcd-io/etcd/blob/b9b7c9328e6f37db1de9a214d414bd2aa02d391e/tools/rw-heatmaps/rw-benchmark.sh#L111-L137

Reply:
Lets implement optional environment variables for these uppercase variables as a basic starting point: https://github.com/etcd-io/etcd/blob/b9b7c9328e6f37db1de9a214d414bd2aa02d391e/tools/rw-heatmaps/rw-benchmark.sh#L5-L26.

Over time I am happy for more command line parameters to be added if we think it will improve user experience but it's not critical right now imo.

Reply:
Agreed. Do you also want one for `CLIENT_PORT`? I initially left that one out.

Reply:
Yeah let's implement environment variable support for all those uppercase vars. I don't see why client port should be hard coded as etcd server allows it to be configurable.

Reply:
Done. I also added a `.gitignore` to address the previous PR's last comment: https://github.com/etcd-io/etcd/pull/17428#pullrequestreview-1926814342.